runtime/doc/builtin.txt

```diff
@@ -4327,6 +4329,27 @@ getregion({pos1}, {pos2} [, {opts}])			*getregion()*
 		Can also be used as a |method|: >
 			getpos('.')->getregion(getpos("'a"))
 <
+getregionpos({pos1}, {pos2} [, {opts}])            *getregionpos()*
+		Returns the list of position from {pos1} to {pos2} from a
+		buffer.
+		See |getregion()| for the arguments and notes.
+		Note: When the selection is blockwise, it returns a pair of
+		positions for every line.
```

Comment:
should it always return the same structure? For non-blockwise, the end pos is not technically "necessary", but have the same structure simplifies the logic needed in consumer logic.

Reply:
OK.  Fixed.